src/ctap/mod.rs

```diff
@@ -244,6 +244,69 @@ fn verify_signature(
     Ok(())
 }
 
+// Returns whether the keepalive was sent, or false if cancelled.
+pub fn send_keepalive_up_needed(
```

Comment:
Not sure how useful it is to move this function when `check_user_presence` is still in `Env`.